docusaurus/docs/reference/metadata.md

```diff
@@ -73,6 +74,16 @@ Plugin dependency. Used to display information about plugin dependencies in the
 | `type`    | string | **Yes**  | Possible values are: `app`, `datasource`, `panel`.                                                                                                                              |
 | `version` | string | **No**   | This property should use a specific, pinned [SemVer](https://semver.org/) version and not a range. If no version is specified, it will resolve to the latest supported version. |
 
+### extensions
+
+In case your plugin depends on an exposed component using `usePluginComponent()`, you need to register the id of the exposed component under here, otherwise it won't work.
```

Comment:
```suggestion
If your plugin depends on an exposed component using `usePluginComponent()`, you must register the id of the exposed component here, otherwise it won't work.
```

Reply:
Perhaps these mentioned functions should link to their info? e.g. https://grafana.com/developers/plugin-tools/reference/ui-extensions#useplugincomponents

Reply:
That's a great idea!